src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientImpl.java

```diff
@@ -21,13 +21,9 @@
 
 import javax.net.ssl.SSLContext;
 
-import org.apache.http.Header;
-import org.apache.http.HttpEntity;
-import org.apache.http.HttpHeaders;
-import org.apache.http.HttpResponse;
-import org.apache.http.HttpStatus;
-import org.apache.http.StatusLine;
+import org.apache.http.*;
```

Comment:
Please revert the use of wildcard import.

Reply:
Please just fix the wildcard and I think it will all be fine. Thx.

Reply:
Totally missed that one. Thanks for all help and input, very much appreciated